debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java

```diff
@@ -223,9 +223,14 @@ static String logMinerContentsQuery(OracleConnectorConfig connectorConfig, Strin
         StringBuilder query = new StringBuilder();
         query.append("SELECT SCN, SQL_REDO, OPERATION_CODE, TIMESTAMP, XID, CSF, TABLE_NAME, SEG_OWNER, OPERATION, USERNAME ");
         query.append("FROM ").append(LOGMNR_CONTENTS_VIEW).append(" ");
-        query.append("WHERE OPERATION_CODE IN (1,2,3,5) ");
-        query.append("AND SCN > ? ");
-        query.append("AND SCN < ? ");
+        query.append("WHERE ");
+        query.append("SCN > ? AND SCN <= ? ");
+        query.append("AND (");
+        // COMMiT/ROLLBACK/MISSING SCN
+        query.append("(OPERATION_CODE IN (7,34,36) AND USERNAME NOT IN (").append(getExcludedUsers(logMinerUser)).append(")) ");
```

Comment:
This should change to capture COMMIT and ROLLBACK even from the logMinerUser, see 5d7faf42a926791eef0362c985da2fa8b8eb697f

Reply:
Thanks @ckamm, I'll push a commit.  Appreciate the quick review.